src/main/config/firecloud-orchestration-compose.yaml.ctmpl

```diff
@@ -15,6 +15,13 @@ app:
     - FIRECLOUD_URL_ROOT={{.Data.env_firecloud_url_root}}
     - GOOGLE_SECRET_JSON={{.Data.env_google_secret_json}}
     - ADMIN_REFRESH_TOKEN={{.Data.env_admin_refresh_token}}
+    - REFRESH_TOKEN_SECRET_JSON={{.Data.env_refresh_token_secret_json}}
```

Comment:
It's not clear to me how all the different credentials interact, so I would like to keep them separate for today, and revisit post-release.